server/src/main/java/org/elasticsearch/index/mapper/SourceToParse.java

```diff
@@ -94,4 +109,8 @@ public XContentType getXContentType() {
     public XContentMeteringParserDecorator getMeteringParserDecorator() {
         return meteringParserDecorator;
     }
+
+    boolean getIncludeSourceOnError() {
```

Comment:
this doesn't need to be public?

Reply:
This is sufficient, it's only ever read from DocumentParser (in the same package)